.github/workflows/lighthouse.yml

```diff
@@ -35,29 +35,28 @@ jobs:
       with:
         site_name: "happy-franklin-69b6d4"
 
-    # lighthouse local test
+    # lighthouse localhost test
     - name: Lighthouse CI
       run: |
         npm install -g @lhci/cli@0.3.x
         lhci autorun --config=.github/lighthouse/lighthouserc.json
-        lhci autorun --config=.github/lighthouse/lighthouserc.json --upload.target=temporary-public-storage --collect.url=${{ steps.waitFor200.outputs.url }} || echo "LHCI failed!"
       env:
         LHCI_GITHUB_APP_TOKEN: ${{ secrets.LHCI_GITHUB_APP_TOKEN }}
 
-    #runs the lighthouse action
-    # - name: Lighthouse CI Action
-    #   uses: treosh/lighthouse-ci-action@v2
-    #   with:
-    #     urls: |
-    #       $DEPLOY_URL
-    #     budgetPath: '.github/lighthouse/budget.json' # test performance budgets
-    #     configPath: '.github/lighthouse/lighthouserc.json'
-    #     # Path to a LHCI lighthouserc.json file
-    #     # Opt-out of saving Lighthouse results as an action artifacts
-    #     uploadArtifacts: false
-    #     # Number of runs to do per URL
-    #     serverBaseUrl: ${{ secrets.LHCI_SERVER_URL }}
-    #     serverToken: ${{ secrets.LHCI_SERVER_TOKEN }}
-    #   env:
-    #     DEPLOY_URL: ${{ steps.waitFor200.outputs.url }}
-    
+    # runs the lighthouse action (deployed)
```

Comment:
I think this config will be overridden later, right?

Reply:
Your work on lhci and netflify testing was merged into master with [PR 367](https://github.com/ambianic/ambianic-ui/pull/367) a few days ago. 

This PR only pulls in your work on vuetify component optimization. Custom amb components and a few new cypress tests.

Reply:
BTW, github labeled the file you are commenting on as outdated. It is not the version in this PR branch.

Reply:
oh i see now

Reply:
oh i see now